gapic/src/main/com/google/gapid/models/Follower.java

```diff
@@ -283,6 +286,8 @@ public default void onStateFollowed(Path.Any path) { /* empty */ }
     public default void onMemoryFollowed(Path.Memory path)  { /* empty */ }
 
     public default void onTextureFollowed(Service.Resource resource) {}
+
+    public default void onFramebufferAttachmentFollowed(Path.Any path) {}
```

Comment:
nit: it looks like we can parse the Path.Any earlier to make type restriction here stricter?

Reply:
Yes, this should accept a `Path.FramebufferAttachment`